sky/serve/serve_utils.py

```diff
@@ -731,10 +732,25 @@ def format_service_table(service_records: List[Dict[str, Any]],
             f'{replica_table}')
 
 
+def _sort_replica_records(
+        replica_records: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+
+    def _sort_key(record: Dict[str, Any]) -> Tuple[int, int, int]:
+        # If no version, then position replica at the end.
+        version = int(record.get('version', -math.inf))
```

Comment:
```python
>>> int(-math.inf)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: cannot convert float infinity to integer
```
I got this error when i try the code. Actually, if no records, that means it is a service launched before the update feature is introduced and all of the replicas should have no version information.

Reply:
Updated to use `-sys.maxsize` instead. If all the replicas have no version information, then it won't affect the sorting order. If there somehow is some replica with version and one without version being compared, we show the replica without version last